addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2774,7 +2774,7 @@ options.registry.anchor = options.Class.extend({
             const title = $titles.length > 0 ? $titles[0].innerText : this.data.snippetName;
             const anchorName = this._text2Anchor(title);
             let n = '';
-            while (document.getElementById(anchorName + n)) {
+            while (this.$target[0].ownerDocument.getElementById(anchorName + n)) {
```

Comment:
Just a question: what is the difference between `this.$target[0].ownerDocument` and `this.ownerDocument` used at the end of this function ? (asking to know if it should be modified too or if what you wrote could be simplified or if we don't care haha) :slightly_smiling_face:

Reply:
They're the same, I sometimes forge that this shortcut exists. 😅 I changed it.